README.md

```diff
@@ -84,7 +84,10 @@ compatibility, use the Node.js streams `pipe()` method.
 
 ## API
 
-### createParseStream()
+### createParseStream(opts)
+
+* `opts` {Object} an options object
+* `opts.multibyte` {Boolean} handle multibyte chars, defaults to true
```

Comment:
Any reason to make this a flag?

Reply:
Not for our use case, but for general usage some folks may not care to pay the overhead for it.